models/cow_protocol/ethereum/cow_protocol_ethereum_trades.sql

```diff
@@ -205,16 +205,10 @@ valued_trades as (
            fee,
            fee_atoms,
            (CASE
-                WHEN sell_price IS NOT NULL THEN
-                    CASE
-                        WHEN buy_price IS NOT NULL and buy_price * units_bought > sell_price * units_sold
-                            then buy_price * units_bought * fee / units_sold
```

Comment:
```
buy_price * units_bought * fee / units_sold
1.001076 * 0.005 * 0.0010148996324193 / 3e-18 = 1693319440706.3
```

From this [TX](https://etherscan.io/tx/0x84d57d1d57e01dd34091c763765ddda6ff713ad67840f39735f0bf0cced11f02)